mysite/code/services/AddonBuilder.php

```diff
@@ -91,7 +91,9 @@ private function buildReadme(Addon $addon, $path) {
                 }
 
                 $parser = GitHubMarkdownService::create();
+                $parser->setContext($addon->Name);
```

Comment:
I think we need to be a bit more clever here. The context is the the `[user]/[repo]` string not just the name of the add on (which could be different) - for example `silverstripe/fulltextsearch` is actually `silverstripe/silverstripe-fulltextsearch` on github.

I think we should check the repo URL is on GH and also determine the repo slug.

If it's not a GH repo  then don't set the context. If the context isn't set the MarkdownService should just use the RAW endpoint (or not set the context if that's the same result)